.github/workflows/ci.yml

```diff
@@ -196,6 +196,12 @@ jobs:
       - name: lint
         run: yarn backstage-cli repo lint --since origin/master
 
+      - name: bootstrap kind
+        run: |
+          curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.14.0/kind-linux-amd64
+          chmod +x ./kind
+          ./kind create cluster
```

Comment:
Thoughts on using https://github.com/helm/kind-action instead? Would also be interesting to check the speed of that one

Reply:
I'm gonna try and use it. Where can I check the speed of each step?

Reply:
Anyway I don't think is gonna be faster since it's doing the exact same things

Reply:
Yep didn't have my hopes up for being faster, but still feels a bit better to use that action.

Coming back to this I'm worried about the CI test speed tbh. A full run is supposed to take around 15 minutes atm, but the tests on this PR take around 25 minutes. My theory would be that the k8s cluster is taking up resources from the test runner, especially memory. The GitHub actions runners are pretty small, so it wouldn't be too surprising if having a cluster in the background causes that big of an impact.

Adding 1 minute to all tests for the setup was unfortunate but somewhat alright, but almost doubling the test run of course isn't. We'll have to explore some alternate setups for these tests, perhaps something similar to https://github.com/backstage/backstage/blob/master/packages/techdocs-cli/e2e-test.config.js?

Reply:
Can you tell me more about the e2e-test config you posted? I cannot see the link with our problem

Reply:
It's a separate Jest config with test files that live in a different package-level directory. That way the tests don't get executed with the regular test run and you can run them separately when needed.

Reply:
Ok, I'm cool with it. Running them in a separate time/workflow should be the way to go, but when are we supposed to run them?

Reply:
I think we can take in inspiration from TechDocs there too tbh: https://github.com/backstage/backstage/blob/df6ac70c897aa9a5704d8cc078c630daa27ad16e/.github/workflows/verify_e2e-techdocs.yml#L4-L9

Reply:
I created a separate workflow verify_kubernetes.yml. Technically it could be shared with other Kubernetes test